arangod/Graph/BaseOptions.cpp

```diff
@@ -614,3 +607,53 @@ void BaseOptions::isQueryKilledCallback() const {
     THROW_ARANGO_EXCEPTION(TRI_ERROR_QUERY_KILLED);
   }
 }
+
+void BaseOptions::setVertexProjections(Projections projections) {
+  _vertexProjections = std::move(projections);
+}
+
+void BaseOptions::setEdgeProjections(Projections projections) {
+  _edgeProjections = std::move(projections);
+}
+
+Projections const& BaseOptions::getVertexProjections() const {
+  return _vertexProjections;
+}
+
+Projections const& BaseOptions::getEdgeProjections() const {
+  return _edgeProjections;
+}
+
+void BaseOptions::toVelocyPackBase(VPackBuilder& builder) const {
+  TRI_ASSERT(builder.isOpenObject());
+  builder.add("parallelism", VPackValue(_parallelism));
+  builder.add(StaticStrings::GraphRefactorFlag, VPackValue(refactor()));
```

Comment:
Do we still need the GraphRefactor flag? Is it needed for 3.9 compatibility?

Reply:
Will be removed in separate PR as it is unrelated.